wix/readme.TXT

```diff
@@ -0,0 +1,37 @@
+Requirements for build environment:
+===================================
+
+1. Windows Installer XML (WiX) toolset, 3.6 or later, http://wixtoolset.org/releases/
+2. Windows SDK for building multi-lingual setups (optional).
+3. Digital signature service if the MSI should be signed (optional).
+
+
+How to upgrade to a new Skype Version:
```

Comment:
I think Skype is a typo here

Reply:
You are right... the file has been copied over and than changed. This is a leftover.